### ISSUE TITLE
Deliver submission Ad ID information Error after 2.62.1 update

### ISSUE THREAD

**Issue by NebulaPark** [fastlane/fastlane#10735]
### New Issue Checklist

- [✓] Updated fastlane to the latest version
- [✓] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [✓] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [✓] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

There is an error in submitting the examination after 2.26.1.

I have written the functions of fastlane each separately and are using it separately for my environment.

The currently used Deliverfile option has been in use for over a year without change,
```app_version "1.0.0"

submit_for_review true
automatic_release true
skip_binary_upload true


app_review_information(
notes: ""
)


force true

submission_information({
    add_id_info_uses_idfa: true,
    add_id_info_tracks_install: true,
    add_id_info_limits_tracking: true,

    add_id_info_serves_ads: false,
    add_id_info_tracks_action: false,
    content_rights_has_rights: false,
    content_rights_contains_third_party_content: false,
    export_compliance_uses_encryption: false,
    export_compliance_contains_proprietary_cryptography: false,
    export_compliance_contains_third_party_cryptography: false,
    export_compliance_encryption_updated: false
})
```

Since the 2.26.1 update, there is an error that can not be submitted for review regarding the Ad ID option.

  `Deprecates add_id_info_limits_tracking in favor of add_id_info_uses_idfa`

I suspect that changes to the update may be relevant, but I do not know the exact reason.

There is no problem if you manually submit.

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you
```
+--------------------------------------+----------------------------------+
|                         deliver 2.62.1 Summary                          |
+--------------------------------------+----------------------------------+
| username                             | **********@**********          |
| run_precheck_before_submit           | false                            |
| app_icon                             | ./fastlane/AppIcon.png           |
| skip_screenshots                     | true                             |
| app_identifier                       | ********************* |
| app_version                          | 1.0.8                            |
| submit_for_review                    | true                             |
| automatic_release                    | true                             |
| skip_binary_upload                   | true                             |
| force                                | true                             |
| submission_information.add_id_info_  | true                             |
| uses_idfa                            |                                  |
| submission_information.add_id_info_  | true                             |
| tracks_install                       |                                  |
| submission_information.add_id_info_  | false                            |
| serves_ads                           |                                  |
| submission_information.add_id_info_  | false                            |
| tracks_action                        |                                  |
| submission_information.content_righ  | false                            |
| ts_has_rights                        |                                  |
| submission_information.content_righ  | false                            |
| ts_contains_third_party_content      |                                  |
| submission_information.export_compl  | false                            |
| iance_uses_encryption                |                                  |
| submission_information.export_compl  | false                            |
| iance_contains_proprietary_cryptogr  |                                  |
| aphy                                 |                                  |
| submission_information.export_compl  | false                            |
| iance_contains_third_party_cryptogr  |                                  |
| aphy                                 |                                  |
| submission_information.export_compl  | false                            |
| iance_encryption_updated             |                                  |
| screenshots_path                     | ./fastlane/screenshots           |
| metadata_path                        | ./fastlane/metadata              |
| edit_live                            | false                            |
| platform                             | ios                              |
| skip_metadata                        | false                            |
| skip_app_version_update              | false                            |
| overwrite_screenshots                | false                            |
| precheck_default_rule_level          | warn                             |
| ignore_language_directory_validatio  | false                            |
| n                                    |                                  |
+--------------------------------------+----------------------------------+

INFO [2017-10-30 14:13:50.51]: Making sure the latest version on iTunes Connect matches '1.0.8' from the ipa file...
INFO [2017-10-30 14:13:52.06]: '1.0.8' is the latest version on iTunes Connect
INFO [2017-10-30 14:13:52.46]: Loading './fastlane/metadata/ko/release_notes.txt'...
INFO [2017-10-30 14:13:52.46]: Loading './fastlane/metadata/ko/promotional_text.txt'...
INFO [2017-10-30 14:13:52.46]: Loading './fastlane/metadata/copyright.txt'...
INFO [2017-10-30 14:13:54.56]: Activating language ko...
INFO [2017-10-30 14:13:58.58]: Uploading metadata to iTunes Connect
INFO [2017-10-30 14:14:00.83]: Successfully uploaded set of metadata to iTunes Connect
INFO [2017-10-30 14:14:02.20]: Uploading app icon...
INFO [2017-10-30 14:14:07.97]: Selecting the latest build...
INFO [2017-10-30 14:14:17.53]: Selecting build 1.0.8 (14)...
INFO [2017-10-30 14:14:19.28]: Successfully selected build
INFO [2017-10-30 14:14:19.28]: Submitting the app for review...
INFO [2017-10-30 14:14:28.95]: Setting 'add_id_info_uses_idfa' to 'true'...
INFO [2017-10-30 14:14:28.95]: Setting 'add_id_info_tracks_install' to 'true'...
INFO [2017-10-30 14:14:28.95]: Setting 'add_id_info_serves_ads' to 'false'...
INFO [2017-10-30 14:14:28.95]: Setting 'add_id_info_tracks_action' to 'false'...
INFO [2017-10-30 14:14:28.95]: Setting 'content_rights_has_rights' to 'false'...
INFO [2017-10-30 14:14:28.95]: Setting 'content_rights_contains_third_party_content' to 'false'...
INFO [2017-10-30 14:14:28.95]: Setting 'export_compliance_uses_encryption' to 'false'...
INFO [2017-10-30 14:14:28.95]: Setting 'export_compliance_contains_proprietary_cryptography' to 'false'...
INFO [2017-10-30 14:14:28.95]: Setting 'export_compliance_contains_third_party_cryptography' to 'false'...
INFO [2017-10-30 14:14:28.95]: Setting 'export_compliance_encryption_updated' to 'false'...

Looking for related GitHub issues on fastlane/fastlane...
Search query: Something wrong with your Ad ID information: ["ITC.apps.idfa.errorMessage.agreeoptout", "ITC.apps.idfa.errorMessage.binaryviolatesterms"].

URL: https://api.github.com/search/issues?q=Something%20wrong%20with%20your%20Ad%20ID%20information:%20[%22ITC.apps.idfa.errorMessage.agreeoptout%22,%20%22ITC.apps.idfa.errorMessage.binaryviolatesterms%22].+repo:fastlane/fastlane
Found no similar issues. To create a new issue, please visit:
https://github.com/fastlane/fastlane/issues/new
Run `fastlane env` to append the fastlane environment to your issue
~/.rvm/gems/ruby-2.4.1/gems/fastlane-2.62.1/spaceship/lib/spaceship/tunes/tunes_client.rb:979:in `send_app_submission': [!] Something wrong with your Ad ID information: ["ITC.apps.idfa.errorMessage.agreeoptout", "ITC.apps.idfa.errorMessage.binaryviolatesterms"]. (RuntimeError)
```

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.13                                       |
| Ruby                        | 2.4.1                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.8.3                           |
| Installation Source         | ~/.rvm/gems/ruby-2.4.1/bin/fastlane         |
| Host                        | Mac OS X 10.13 (17A405)                     |
| Ruby Lib Dir                | ~/.rvm/rubies/ruby-2.4.1/lib                |
| OpenSSL Version             | OpenSSL 1.1.0f  25 May 2017                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 8.3.3                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | ko_KR.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "1.91.0"

default_platform :ios

platform :ios do
  before_all do
    # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
    
    
  end

  desc "Runs all the tests"
  lane :test do
    scan
  end

  desc "Submit a new Beta Build to Apple TestFlight"
  desc "This will also make sure the profile is up to date"
  lane :beta do
    # match(type: "appstore") # more information: https://codesigning.guide
    gym # Build your app - more options available
    pilot

    # sh "your_script.sh"
    # You can also use other beta testing services here (run `fastlane actions`)
  end

  desc "Deploy a new version to the App Store"
  lane :appstore do
    # match(type: "appstore")
    # snapshot
    gym # Build your app - more options available
    deliver(force: true)
    # frameit
  end

  # You can define as many lanes as you want

  after_all do |lane|
    # This block is called, only if the executed lane was successful

    # slack(
    #   message: "Successfully deployed new App Update."
    # )
  end

  error do |lane, exception|
    # slack(
    #   message: exception.message,
    #   success: false
    # )
  end
end


# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
# All available actions: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md

# fastlane reports which actions are used
# No personal data is recorded. Learn more at https://github.com/fastlane/enhancer

```
</details>

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.62.1  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.2        |
| executable-hooks          | 1.3.2        |
| bundler-unload            | 1.0.2        |
| rubygems-bundler          | 1.4.4        |
| bundler                   | 1.16.0.pre.3 |
| io-console                | 0.4.6        |
| CFPropertyList            | 2.3.5        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.2        |
| babosa                    | 1.0.2        |
| claide                    | 1.0.2        |
| colored                   | 1.2          |
| colored2                  | 3.1.2        |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| dotenv                    | 2.2.1        |
| excon                     | 0.59.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 0.13.1       |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.12.2       |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| jwt                       | 2.1.0        |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.2       |
| logging                   | 2.2.2        |
| memoist                   | 0.16.0       |
| os                        | 0.9.6        |
| signet                    | 0.8.1        |
| googleauth                | 0.6.1        |
| httpclient                | 2.8.3        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.1        |
| google-api-client         | 0.13.6       |
| json                      | 2.1.0        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| plist                     | 3.3.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| slack-notifier            | 1.5.1        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.0        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.5.1        |
| word_wrap                 | 1.0.0        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.3        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| xcpretty-travis-formatter | 1.0.0        |
</details>

*generated on:* **2017-10-30**
</details>

**Comment by NebulaPark** [comment]
This is not code signing  issue

**Comment by NebulaPark** [comment]
This error has been fixed in # 10715

Thank you!

**Comment by minuscorp** [comment]
We're waiting for be reviewed and released. Sorry for the inconvenience @NebulaPark!

**Comment by MaximusMcCann** [comment]
Not fixed in 2.63.0 :/

**Comment by minuscorp** [comment]
Still being reviewed mates! check #10715 for progress. 🚀
